### PR TITLE
feat: add safe, phpstan and psalm

### PR DESCRIPTION
**Pull request by asbiin**

### CODE REVIEW

File: app/Services/CreateLicenceKey.php [hunk 1]
```diff
@@ -62,9 +67,7 @@ public function execute(mixed $payload): LicenceKey
      */
     private function generateKey(): void
     {
-        $this->key = collect();
-
-        $this->key->push([
+        $this->key = collect([
```
Comment (asbiin) [inline app/Services/CreateLicenceKey.php:70]: I've simplified it, there is no need to encapsulate it in 2 arrays

**Review by djaiss — APPROVED** [review]
Good job!

**Comment by djaiss** [comment]
This PR is now in error but I haven't changed anything that would make this PR to be in error now :-/

File: app/Services/DestroyLicenceKey.php [hunk 2]
```diff
@@ -2,10 +2,12 @@
 
 namespace App\Services;
 
+use App\Models\LicenceKey;
```
Comment (asbiin) [inline app/Services/DestroyLicenceKey.php:5]:
```suggestion
```

File: app/Services/RenewLicenceKey.php [hunk 3]
```diff
@@ -2,10 +2,12 @@
 
 namespace App\Services;
 
+use App\Models\LicenceKey;
```
Comment (asbiin) [inline app/Services/RenewLicenceKey.php:5]:
```suggestion
```

**Pull request merged**